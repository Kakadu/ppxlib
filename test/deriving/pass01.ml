type t = int [@@deriving foo, bar]
