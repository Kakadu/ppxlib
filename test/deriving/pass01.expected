type t = int[@@deriving (foo, bar)]
let _ = fun (_ : t)  -> () 
let x = 42 
let _ = x 
let () = Printf.printf "x = %d\n" x 
